### PR TITLE
Fixed Missing support for `include_[ATTRIBUTE]?`

### PR DESCRIPTION
**Pull request by swatijadhav**
Hello,

Here I have added some changes to have support for  `include_[ATTRIBUTE]?`

https://github.com/dockyard/postgres_ext-serializers/issues/26

Let me know if any changes needed. Thanks.

### CODE REVIEW

**Comment by danmcclain** [comment]
Could you please add tests?

File: lib/postgres_ext/serializers/active_model/array_serializer.rb [hunk 1]
```diff
@@ -50,7 +50,9 @@ def _include_relation_in_root(relation, *args)
 
       _serializer = serializer_class.new klass.new, options
 
-      attributes = serializer_class._attributes
+      attributes = serializer_class._attributes.select{ |key, value|
+        _serializer.send("include_#{key}?") }
```
Comment (danmcclain) [inline lib/postgres_ext/serializers/active_model/array_serializer.rb]: Please use `do ... end` with multiline blocks

**Comment by swatijadhav** [comment]
@danmcclain Yeah Sure.

**Comment by swatijadhav** [comment]
@danmcclain I have added test cases for the changes. Could you please review it and let me know if any changes needed.

Thanks :smile:

File: test/sideloading_test.rb [hunk 2]
```diff
@@ -143,4 +143,26 @@
...
+
+  context 'support for include_[attrbute]' do
+    let(:relation)   { User.all }
+    let(:controller) { UserController.new }
+    let(:options)    { { each_serializer: UserSerializer } }
+
+    before do
+      user    = User.create name: 'John', mobile: "51111111"
+      address = Address.create district_name: "mumbai", user_id: user.id
+      @json_expected = "{\"users\":[{\"id\":#{user.id},\"name\":\"John\",\"mobile\":\"51111111\",\"offer_ids\":[],\"reviewed_offer_ids\":[]}],\"offers\":[],\"addresses\":[{\"id\":#{address.id},\"district_name\":\"mumbai\"}]}"
```
Comment (danmcclain) [inline test/sideloading_test.rb]: You might as well just have these in the `it` blocks, as they are only used once each

**Comment by danmcclain** [comment]
Once you fix that comment, could you squash your commits? Thanks :+1:

File: test/sideloading_test.rb (hunk 2 above)
Comment (swatijadhav) [inline test/sideloading_test.rb]: Thanks for reviewing changes. I will add the changes as you suggested and also squash commits. :+1:

**Comment by swatijadhav** [comment]
@danmcclain Changes are done as per your suggestion. Thanks :+1:

**Pull request merged**